develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -0,0 +1,381 @@
+---
+title: Logs
+sidebar_order: 3
+---
+
+<Alert title="Note" level="warning">
+
+The Sentry Logs feature is under active development. The information in this document is subject to change.
+
+</Alert>
+
+This document defines the format used by Sentry to ingest logs, as well as the SDK API and behavior that create and send logs to Sentry.
+
+## Logs Protocol
+
+There are two ways to send logs to Sentry: via the `log` envelope and Sentry Log protocol (preferred), or the `otel_log` envelope and OpenTelemetry Log protocol.
+
+### `otel_log` Envelope Item Payload
+
+The `otel_log` envelope item payload is a JSON object that represents an OpenTelemetry Log. Multiple `otel_log` envelope items can be sent in a single envelope.
+
+```json
+{
+  "severity_text": "info",
+  "body": {
+    "string_value": "User John has logged in!"
+  },
+  "attributes": [
+    {
+      "key": "sentry.message.template",
+      "value": { "string_value": "User %s has logged in!" }
+    },
+    {
+      "key": "sentry.message.parameters.0",
+      "value": { "string_value": "John" }
+    }
+  ],
+  "time_unix_nano": "1741191250694000000",
+  "trace_id": "edec519707974fc8bfccb5a017e17394",
+  "severity_number": 9
+}
+```
+
+It consists of the following fields:
+
+`severity_text`
+
+: **String, required**. The severity level of the log. One of `trace`, `debug`, `info`, `warn`, `error`, `fatal` (in order of lowest to highest).
+
+`severity_number`
+
+: **Integer, optional**. The severity number of the log. See [Log Severity Number](#log-severity-number) for more information.
+
+`trace_id`
+
+: **Integer, optional**. [HEAVILY RECOMMENDED] The trace id of the log. SDKs should attempt to set this if possible. This is used to link logs to traces and errors in Sentry.
```

Comment:
Why not make it required? Every SDK should support [tracing without performance](https://develop.sentry.dev/sdk/telemetry/traces/tracing-without-performance/) already, and for that you need a trace ID.

Reply:
(moved from internal doc): Can we use DSC for this? Or is the incentive here that multiple log envelope items within a single envelope can belong to different traces?

Reply:
This is optional in OTel since people may want to do various things, but for our product we always expect a trace. Therefore for our own protocol I think it should be required, if you don't have a traceid it's fine to make one (like with tracing without perf, as Philipp mentioned).

Reply:
alright, but what about the DSC? I guess it also has to be a part of the logs envelope right? What trace_id goes in there?

Reply:
For now DSC is not part of the logs envelope - just trace, given the behaviour that logs from multiple traces might be in a single envelope.

Once we need DSC behaviour we'll revisit this.